IFC4x3/Properties/p/PhaseNumber_0ZOkcOH$14AAoaednCd9ms/Documentation.md

```diff
@@ -0,0 +1 @@
+Number of phase that the equipment operates on.
```

Comment:
Just checking out of ignorance: is it number of phases or a specification of which phase?

Reply:
I have changed the property to "NumberOfPhase" to make it more understandable.
0871f7b28802c8b6bf8a2b767b0356cb11e25edd